### PR TITLE
Restore PHP<8 backward compatibility

### PR DESCRIPTION
**Pull request by lufton**

### CODE REVIEW

File: exchange/orders.php [hunk 1]
```diff
@@ -184,9 +184,10 @@ function wc1c_replace_document_services($order, $document_services) {
   }
 
   if (!$shipping_methods) {
-    $shipping = WC()?->shipping;
-    $shipping?->load_shipping_methods();
-    $shipping_methods = $shipping?->get_shipping_methods();
```
Comment (lufton) [inline exchange/orders.php:189]: This is just a syntax sugar witch breakes PHP backward compatibility.

Comment (Alex01d) [reply exchange/orders.php:189]: Thanks, that's from Rector

**Comment by Alex01d** [comment]
Also `str_starts_with` and `str_contains` should break compatibility.

**Comment by sgtpep** [comment]
Fyi, @Alex01d and @lufton, I reverted the changes introduced in #72. It was a premature decision to merge it to `master` as it indeed breaks backward compatibility with earlier PHP versions, and we shouldn't break it. Sorry for the trouble!

**Pull request merged**